docs/ChangeLog/20191129/PR6825.md

```diff
@@ -0,0 +1,4 @@
+# Update LUFA submodule
```

Comment:
Would you please move/rename this file to `docs/ChangeLog/20200229/PR6825.md`?

Do that and I'll merge this into `future` branch.